CRM/Core/BAO/CustomGroup.php

```diff
@@ -28,6 +28,83 @@ public static function self_hook_civicrm_post(\Civi\Core\Event\PostEvent $e): vo
     Civi::cache('metadata')->flush();
   }
 
+  /**
+   * Master function to retrieve custom groups with their fields
+   *
+   * This provides a cached list of all custom groups, with a nested array of fields.
+   *
+   * @param bool $includeDisabled
+   *   If false (default) this will filter out disabled groups and fields.
+   * @return array[]
+   */
+  public static function getAll(bool $includeDisabled = FALSE): array {
```

Comment:
So I would call this `getActive()` and split out `getAll()`  to be a function called from it - my experience is that once `getAll() has parameters then you get more added to it....

Reply:
That makes sense.